src/NumberTheory/QuadFormAndIsom/enumeration.jl

```diff
@@ -941,9 +941,22 @@ function representatives_of_hermitian_type(
       return reps
     end
     # local conditions are okay, enumerate the genus
-    allow_info && println("Enumerate Z-genus $G")
+    allow_info && !_local && println("Enumerate Z-genus $G")
     repre = oscar_genus_representatives(G; genusDB, root_test, info_depth, max_lat=first ? 1 : inf, _local)
-    allow_info && println("$(length(repre)) representative(s)")
+    if root_test && _local && signature_pair(G)[1]==0 && numerator(mass(G))==1 && rank(G)<=12
+      if mass(G) == 1//automorphism_group_order(T)
```

Comment:
```suggestion
    if root_test && _local && signature_pair(G)[1]==0 && rank(G)<=12
      if numerator(mass(G))==1 && mass(G) == 1//automorphism_group_order(T)
```
I imagine that it is what you wanted to do